src/api/routes/tokens.ts

```diff
@@ -228,5 +229,25 @@ export function createTokenRouter(db: PgStore): express.Router {
     })
   );
 
+  router.get(
+    '/ft/holders/:token',
```

Comment:
I think `/ft/:token/holders` would be a better path, but I'm ok if you prefer to leave it as is

Reply:
+1 seems more future proof as well

Reply:
Agreed! Updated